### PR TITLE
Fix async hang

### PR DESCRIPTION
**Pull request by DesmonDay**
### PR types
Bug fixes

### PR changes
Others

### Descriptions
Fix mtime flag in rotate_checkpoints.

### CODE REVIEW

**Review by wawltor — DISMISSED** [review]
LGTM

**Review by DrownFish19 — DISMISSED** [review]
LGTM

File: paddlenlp/trainer/auto_trainer.py [hunk 1]
```diff
@@ -687,7 +687,13 @@ def _save_checkpoint(self, model, metrics=None):
...
                 paddle.save(self.state.global_step, os.path.join(output_dir, ".checkpoint_done"))
 
-    def _save(self, output_dir: Optional[str] = None, state_dict=None, merge_tensor_parallel=False):
+    def _save(
+        self,
+        output_dir: Optional[str] = None,
+        state_dict=None,
+        merge_tensor_parallel=False,
+        signal_dir: Optional[str] = None,
+    ):
```
Comment (ZHUI) [inline paddlenlp/trainer/auto_trainer.py:696]: 接口变了吗，自动并行也得改？

Comment (DesmonDay) [reply paddlenlp/trainer/auto_trainer.py:696]: 嗯嗯

**Pull request merged**